cnf-certification-test/webserver/index.js

```diff
@@ -211,4 +207,134 @@ var field='<label for="'+clickedId+counters[clickedId] +'">'+counters[clickedId]
  }
  counters[clickedId]= counters[clickedId]+1
 }
+}
+
+ function addCheckedTest(){
+  document.getElementById('lifecycle').checked = true
+  document.getElementById('manageability').checked = true
+  document.getElementById('affiliated-certification').checked = true
+  document.getElementById('operator').checked = true
+  document.getElementById('access-control').checked = true
+  document.getElementById('platform-alteration').checked = true
+  document.getElementById('networking').checked = true
+  document.getElementById('performance').checked = true
+  document.getElementById('observability').checked = true
+
+ } 
+
+function selectScenario(){
+  
+  $('#testTable').empty()
+  var beginning='<caption> Select a Suite Name </caption> <colgroup> '+
+  '<col style="width: 200px;">'+
+   '<col>'+
+    '</colgroup>'+
+  '<thead><tr> <th style="width: 200px;" scope="col" data-label="Testname">Test Name<rh-sort-button></rh-sort-button></th>'+
+    '<th style="width: 500px;" scope="col" data-label="Description">Test Info</th>'+
+    '</tr></thead>'
+    $(beginning).appendTo('#testTable');
+
+  addCheckedTest()
+  var field = ""
+  const selectScenarioComboBox = document.getElementById('selectScenarioComboBox')
+  const selectedValue = selectScenarioComboBox.options[selectScenarioComboBox.selectedIndex].value
+  if (selectedValue === 'all') {
+    document.getElementById('selectOpt').setAttribute('hidden','hidden')
+
+  for (const key in classification) {
+    field='<tr id="'+key+'"><td data-label="Testname"><label><input type="checkbox" value="'+
+     key +'" name="selectedOptions" checked> ' +
+    '<h7 for="'+key+'">'+key+
+    '</h7></label></td>'+
+    '<td data-label="Description"><rh-accordion>'+
```

Comment:
The accordion is taking too much vertical space. Each test should be taking only one line of text (as thin as possible). On the line I would put a checkbox, the test name and the test description (whatever fits). Inside expanded accordion, put the rest of the info: full test description, remediation, clickable link to best practice, etc...
![image](https://github.com/test-network-function/cnf-certification-test/assets/86730676/d0e83858-dc5d-4c42-88cc-912f5191be4f)

Reply:
I would also make all tests always visible, the only changes are what tests are selected when selecting the test tag or suite. We should have also a "none" scenario  to start with no test case selected
To make it easier to see the tests per suite, we could use:
- tabs per suite (suite checkbox in tab to select all tests in suite)
- an accordion per suite (also suite checkbox in accordion), which will contain the accordion per test
- a header /block that contains all test from a suite

Reply:
Add a counter of selected tests

Reply:
the latest seems to be missing a overall "none" scenario. 
Also all tests should be always visible, not hidden depending on scenario. when switching scenarion the tests should be selected/un-selected
It would be nice to have a total selected test counter as well, not just per suites